_dev/src/views/configuration.vue

```diff
@@ -17,125 +17,177 @@
  * International Registered Trademark & Property of PrestaShop SA
  *-->
 <template>
-  <spinner v-if="loading && !showPopupGlass && !showTokensGlass" />
-  <div
-    id="configuration"
-    class="ps-facebook-configuration-tab"
-    v-else
-  >
-    <introduction
-      v-if="!psAccountsOnboarded && showIntroduction"
-      @onHide="showIntroduction = false"
-      class="m-3"
-    />
-    <MultiStoreSelector
-      v-else-if="!contextPsAccounts.isShopContext && shops.length"
-      :shops="shops"
-      class="m-3"
-      @shop-selected="onShopSelected($event)"
-    />
-    <template v-else>
-      <messages
-        :show-onboard-succeeded="psFacebookJustOnboarded"
-        :show-sync-catalog-advice="psAccountsOnboarded && showSyncCatalogAdvice"
-        :category-matching-started="categoryMatchingStarted"
-        :product-sync-started="productSyncStarted"
-        :ad-campaign-started="adCampaignStarted"
-        :alert-settings="alertSettings"
-        @onSyncCatalogClick="onSyncCatalogClick"
-        @onCategoryMatchingClick="onCategoryMatchingClick"
-        @onAdCampaignClick="onAdCampaignClick"
+  <div>
```

Comment:
This is one of the change of this file, the rest is being the indentation and the modal being added at the end.

Ignoring whitespace-only changes may help to review.